modules/pel/peltool/README.md

```diff
@@ -0,0 +1,83 @@
+# peltool Usage
+
+To see all of peltool's capabilities, use the help option `peltool.py --help`.
+
+**Note:** For non-BMC environment user must pass the PEL directory to parse PELs
+          e.g. `peltool.py -p <pel_dir> -l`, `peltool.py -p <pel_dir> -n`,
+               `peltool.py -p <pel_dir> -a`
+
+#### List PELs
+
+- Get list of PELs: `peltool.py -l` (By default serviceable PELs are listed)
+- Get list of PELs including serviceable PELs: `peltool.py -ls`
```

Comment:
I thought you were going to remove this line?  since serviceable is the default you don't need to pass it in.